test/name-sync-test.js

```diff
@@ -53,35 +50,6 @@ describe('tmp', function () {
           }
         });
       });
-      describe('on issue #246', function () {
```

Comment:
Could you leave these tests and make these so that they check for the presence of the single quotes and double quotes instead?

Reply:
Absolutely! I wasn't sure which you would have preferred between simply reverting or keeping the tests but amended to serve as regression tests. I'll do this when I can, either tonight or tomorrow.

Reply:
Please see [a831713](https://github.com/raszi/node-tmp/pull/278/commits/a831713743a1e370ebcda62f0709fdda9c0dd9d6).